virttest/qemu_vm.py

```diff
@@ -891,29 +891,34 @@ def add_memorys(devices, params):
 
             :param devices: VM devices container
             """
-            options = []
-            mem_params = params.object_params("mem")
-            mem = float(mem_params.get("mem", 512))
-            if mem_params.get("maxmem"):
-                maxmem = float(utils_misc.normalize_data_size(mem_params.get("maxmem"), order_magnitude="M"))
-                actual_mem = int(min(mem, maxmem))
-            else:
-                actual_mem = int(mem)
-            params["mem"] = actual_mem
-            options.append("%s" % params["mem"])
-            if mem_params.get("slots") and mem_params.get("maxmem"):
-                options.append("slots=%s" % mem_params["slots"])
-                options.append("maxmem=%s" % mem_params["maxmem"])
-            if not options:
-                return devices
+
+            if params.get("maxmem"):
+                normalize_data_size = utils_misc.normalize_data_size
+                values = map(
```

Comment:
not care it, utils_misc.normalize_data_size can resolve this issue. append 'M' to avoid not unit given case.